python/tank/platform/engine.py

```diff
@@ -2092,17 +2092,115 @@ def _initialize_dark_look_and_feel(self):
         designed to work well with most dark themes.
         
         However, if you are for example creating your own QApplication instance
-        you can execute this method to but the session into Toolkit's 
+        you can execute this method to put the session into Toolkit's 
         standard dark mode.
         
-        This will initialize the plastique style and set it up with a standard
-        dark palette and supporting stylesheet.
+        This will initialize the plastique style (for Qt4) or the fusion style
+        (for Qt5), and set it up with a standard dark palette and supporting
+        stylesheet.
         
         Apps and UIs can then extend this further by using further css.
         
         Due to restrictions in QT, this needs to run after a QApplication object
         has been instantiated.
         """
+        if self.has_qt5:
+            self.__initialize_dark_look_and_feel_qt5()
+        else:
```

Comment:
what happens if we don't have QT? Just for protection, should this if be something like this?

```
if self.has_qt5:
   xxx
elif self.has_qt4:
   xxxx
else:
   log.warning("cannot init because no qt")

Reply:
Good catch.